src/messages/messages_fr.properties

```diff
@@ -1,83 +1,90 @@
-noAccount=Nouvel utilisateur ?
-invalidUserMessage=Nom d''utilisateur ou mot de passe invalide.
-registerTitle=S''enregistrer
-backToLogin=&laquo; Retour à la connexion
-requiredFields=Champs requis
-username=Nom d''utilisateur
 passwordConfirm=Confirmation du mot de passe
-email=Courriel
+email=Email
+name=Nom
 country=Pays
-missingUsernameMessage=Veuillez entrer votre nom d''utilisateur.
-missingPasswordMessage=Veuillez entrer votre mot de passe.
-missingEmailMessage=Veuillez entrer votre courriel.
-error-invalid-value=Valeur invalide.
-error-pattern-no-match=Valeur invalide.
-invalidPasswordMinLengthMessage=Mot de passe invalide : longueur minimale requise de {0}.
-invalidPasswordConfirmMessage=Le mot de passe de confirmation ne correspond pas.
-usernameExistsMessage=Le nom d''utilisateur existe déjà.
-emailExistsMessage=Le courriel existe déjà.
-invalidEmailMessage=Courriel invalide.
-doSignOut=Déconnexion
-signOut=Déconnexion
-personalInfo=Information personnelle :
-personalSubMessage=Gérez ces informations de base: votre prénom, nom de famille et email
-selectALocale=Sélectionner une langue
-doSave=Sauvegarder
-save=Sauvegarder
+missingUsernameMessage=Please specify username.
+missingPasswordMessage=Please specify password.
+missingEmailMessage=Please specify email.
+error-invalid-value=Invalid value.
+error-pattern-no-match=Invalid value.
+invalidPasswordMinLengthMessage=Invalid password\: minimum length {0}.
+invalidPasswordConfirmMessage=Password confirmation doesn't match.
+usernameExistsMessage=Username already exists.
+emailExistsMessage=Email already exists.
+invalidEmailMessage=Invalid email address.
+
+doSignOut=Se déconnecter
+signOut=Se déconnecter
+
+personalInfo=Personal info
+personalSubMessage=Manage your basic information.
+selectALocale=Select a locale
+doSave=Enregistrer
+save=Enregistrer
 doCancel=Annuler
 cancel=Annuler
 deleteAccount=Supprimer le compte
-deleteAccountWarning=Ceci est irréversible. Toutes vos données seront définitivement supprimées, et irrécupérables
+deleteAccountWarning=This is irreversible. All your data will be permanently destroyed, and irretrievable.
 delete=Supprimer
-accountSecurity=Sécurité du compte
-signingIn=Connexion
-signingInDescription=Configurez les façons de vous authentifier.
-basic-authentication=Authentification de base
-password-help-text=Connectez-vous en saisissant votre mot de passe.
-credentialCreatedAt=<0>Créé</0> {{date}}.
-update=Mettre à jour
-two-factor=Authentification à deux facteurs
-otp-display-name=Application d''authentification
-otp-help-text=Entrez un code de vérification depuis votre application d''authentification.
-setUpNew=Configurer {{name}}
-notSetUp={{name}} n''est pas configuré.
-deviceActivity=Activités des appareils
-signedInDevicesExplanation=Déconnectez-vous de tout appareil non familier.
-signedInDevices=Appareils connectés
-refreshPage=Rafraichir la page
-currentSession=Session actuelle
+accountUpdatedMessage=Your account has been updated.
```

Comment:
@teolemon , do you have any idea what is going on here as this PR seems to be reverting some of the French translations back to English?

Reply:
yes, translations need to be uploaded to crowdin. Currently, French might be empty on Crowdin, and we need to one time upload them there.

Reply:
So is this because I added some more translations in the repo rather than in Crowdin? As we use more aspects of Keycloak new properties will be added and the code is set up to initially use the Keycloak translation for these terms where there is one, which will again be pushed from the repo. Is that going to be a problem?

Reply:
![image](https://github.com/user-attachments/assets/17e4d592-4316-4381-acaa-2fb15e5dea83)
actually no, it seems some translations have been uploaded. They should be commited back as they go back down to GitHub @john-gom